lib/twofishes/result.rb

```diff
@@ -36,21 +24,21 @@ def lng
       feature.geometry.center.lng
     end
 
-    def coordinates
-      [lat, lng]
+    def country_code
+      feature.cc
     end
 
-    def parents
-      @data.parents.map { |parent| Twofishes::Result.new(parent) }
+    def coordinates
+      [lat, lng]
     end
 
-    def method_missing(method_sym, *arguments, &block)
-      @data.send(method_sym)
+    def method_missing(name, *args, &block)
```

Comment:
Why `method_missing` again?

Now all fields that are included in response are known (from thrift output files), could it be implemented using `Forwardable`?